import-export-cli/cmd/exportAPIs.go

```diff
@@ -98,32 +99,37 @@ func executeExportAPIsCmd(exportDirectory string) {
 	exportAPIs()
 }
 
+func ExecuteExportAPIsCmdByDeprecatedCommand(exportDirectory string, data map[interface{}]interface{}) {
```

Comment:
Any reason why we are calling package cmd logic via a deprecated function? Ideally we want to have deprecated functionality separate from from the recommended command logic and this seems to create a dependency.

Reply:
Fixed via https://github.com/wso2/product-apim-tooling/pull/515/commits/996ed73fa64fc162ff3fdfb678e17d81d9af8993